gsoc/2009.html

```diff
@@ -49,7 +49,7 @@
         <li><a href="/index.html#projects">Projects</a></li>
         <li><a href="/index.html#internship-opportunities">Internship Opportunities</a></li>
         <li><a href="/index.html#welcome-newcomers">Welcome Newcomers</a></li>
-        <li class="toggle"><a href="/index.html#main-menu">About Us <span class="fas fa-chevron-down" "arrow"></i></a>
```

Comment:
I think that ```<i>``` tag should not be there either or the closed ```</i>``` is mistyped and it should be ```</li>``` after ```</a>```. And ```</span>``` is also missing, this ambiguity is in each of the files changed.